.github/workflows/integration.yaml

```diff
@@ -51,6 +51,7 @@ jobs:
      timeout-minutes: 30
      strategy:
        max-parallel: 15
+       fail-fast: false
```

Comment:
Looks unrelated.

Reply:
This is so that we can see all the tests at once. It's not related - it's just to allow faster iteration.